src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -139,11 +140,12 @@ public KubernetesCloud(@NonNull String name, @NonNull KubernetesCloud source) {
         this.containerCap = source.containerCap;
         this.retentionTimeout = source.retentionTimeout;
         this.connectTimeout = source.connectTimeout;
+        this.usageRestricted = source.usageRestricted;
     }
 
     @Deprecated
     public KubernetesCloud(String name, List<? extends PodTemplate> templates, String serverUrl, String namespace,
-            String jenkinsUrl, String containerCapStr, int connectTimeout, int readTimeout, int retentionTimeout) {
+            String jenkinsUrl, String containerCapStr, int connectTimeout, int readTimeout, int retentionTimeout, boolean usageRestricted) {
```

Comment:
Not needed (since you have `@DataBoundSetter` on the setter method) and it would break potential usages of this constructor.

Reply:
done